elements/campaign-display/campaign-display.js

```diff
@@ -23,7 +23,7 @@ class CampaignDisplay extends BulbsElement {
   render () {
     return (
       <CampaignDisplayRoot
-          data={this.state}
+          {...this.state}
```

Comment:
This seems like unnecessary indirection, can we make this `campaign={this.state.campaign}` instead? Or is there a specific reason to keep it as a generic unpacking?